### PR TITLE
Pr delete and tag

### PR DESCRIPTION
**Pull request by AiwassPrime**

### CODE REVIEW

**Review by yhtMinceraft1010X — COMMENTED** [review]
From a documentation and logic perspective, LGTM

**Review by QY-H00 — COMMENTED** [review]
LGTM for test part.

**Comment by umergta** [comment]
no changes for storage. LGTM!

File: src/test/java/seedu/module/logic/commands/TagCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        expectedModel.setTask(firstTask, editedTask);
+
+        assertCommandSuccess(tagCommand, model, expectedMessage, expectedModel);
+    }
+
+    @Test
+    void execute_addTagFilteredList_success() {
+        showTaskAtIndex(model, INDEX_FIRST_TASK);
+        Task firstTask = model.getFilteredTaskList().get(INDEX_FIRST_TASK.getZeroBased());
+        String[] expectedTags = new String[firstTask.getTags().size() + 1];
```
Comment (figo2127) [inline src/test/java/seedu/module/logic/commands/TagCommandTest.java]: Maybe assign Set<Tag> tags = firstTask.getTags(), to make code more readable. But not a big issue.

Comment (figo2127) [reply src/test/java/seedu/module/logic/commands/TagCommandTest.java]: LGTM!

Comment (AiwassPrime) [reply src/test/java/seedu/module/logic/commands/TagCommandTest.java]: Nice suggestion, updated!

File: src/main/java/seedu/module/logic/parser/TagCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import seedu.module.model.tag.Tag;
+
+/**
+ * Parses input arguments and creates a new TagCommand object
+ */
+public class TagCommandParser implements Parser {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the TagCommand
+     * and returns an TagCommand object for execution.
```
Comment (umergta) [inline src/main/java/seedu/module/logic/parser/TagCommandParser.java]: should be "returns a"
Just a small fix

Comment (AiwassPrime) [reply src/main/java/seedu/module/logic/parser/TagCommandParser.java]: Fixed.

**Pull request merged**